pkg/scheduler/framework/v1alpha1/interface.go

```diff
@@ -135,6 +139,16 @@ type PrefilterPlugin interface {
 	Prefilter(pc *PluginContext, p *v1.Pod) *Status
 }
 
+// ScorePlugin is an interface that must be implemented by "score" plugins to rank
+// nodes that passed the filtering phase.
+type ScorePlugin interface {
+	Plugin
+	// Score is called on each filtered node. It must return success and an integer
+	// indicating the rank of the node. All scoring plugins must return success or
+	// the pod will be rejected.
+	Score(pc *PluginContext, p *v1.Pod, nodeName string, meta interface{}) (*Status, int)
```

Comment:
We should not have meta in the args. metadata can be stored in PluginContext and used by the plugins that may need it.

Reply:
Done.